src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/CombinationItemType.php

```diff
@@ -98,12 +98,16 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'icon' => 'mode_edit',
                 'attr' => [
                     'class' => 'edit-combination-item tooltip-link',
+                    'data-toggle' => 'pstooltip',
+                    'data-original-title' => $this->trans('Edit item', 'Admin.Actions'),
```

Comment:
```suggestion
                    'data-original-title' => $this->trans('Edit', 'Admin.Global'),
```
I think it's better to use the existing string 'Edit' to avoid translators retranslating it again just for the 'item' part.